### PR TITLE
Education letters static page login update

### PR DESCRIPTION
**Pull request by FernandoBoza**
## Summary
Fixed some typos, UI refactoring and for education letters landing page, reverted to local loginWidget.

## Testing done
- [ ] Done

## What areas of the site does it impact?
http://localhost:3001/education/download-letters/

## Acceptance criteria
- Sign in title is: Please sign in to get your VA education letter.
- Static page, view-education-letters-login, now toggles UI depending on visitor vs user logged in.

### CODE REVIEW

**Review by cesaresantiago — APPROVED** [review]

File: src/applications/education-letters/components/LoginWidget.jsx [hunk 1]
```diff
@@ -1,56 +1,64 @@
...
+    <va-alert
+      close-btn-aria-label="Close notification"
+      status="continue"
+      visible
+    >
+      <p
+        className="vads-u-font-size--h3 vads-u-font-weight--bold vads-u-font-family--serif"
+        slot="headline"
+      >
+        Please sign in to get your VA education letter.
```
Comment (btruongafs) [inline src/applications/education-letters/components/LoginWidget.jsx:21]: Update to "Sign in to download your VA education decision letter"

File: src/applications/static-pages/view-education-letters-login/LoginWidget.jsx [hunk 2]
```diff
@@ -18,7 +23,7 @@ const LoginInWidget = ({ toggleLoginModal }) => {
         className="vads-u-font-size--h3 vads-u-font-weight--bold vads-u-font-family--serif"
         slot="headline"
       >
-        Please sign in to verify your VA education letter.
+        Please sign in to get your VA education letter.
```
Comment (btruongafs) [inline src/applications/static-pages/view-education-letters-login/LoginWidget.jsx:26]: Update to "Sign in to download your VA education decision letter"

File: src/applications/static-pages/view-education-letters-login/LoginWidget.jsx [hunk 3]
```diff
@@ -49,16 +54,55 @@ const LoginInWidget = ({ toggleLoginModal }) => {
...
+    </div>
+  );
+
+  const loggedInUserUI = (
+    <button
+      className="va-button-primary"
+      type="button"
+      onClick={redirectToEducationLetters}
+    >
+      Get your education letter
```
Comment (btruongafs) [inline src/applications/static-pages/view-education-letters-login/LoginWidget.jsx:74]: Update to "Download your VA education decision letter"

File: src/applications/education-letters/components/LoginWidget.jsx [hunk 4]
```diff
@@ -1,56 +1,64 @@
...
+    <va-alert
+      close-btn-aria-label="Close notification"
+      status="continue"
+      visible
+    >
+      <p
+        className="vads-u-font-size--h3 vads-u-font-weight--bold vads-u-font-family--serif"
+        slot="headline"
+      >
+        Download your VA education decision letter
```
Comment (btruongafs) [inline src/applications/education-letters/components/LoginWidget.jsx:21]: This should be "Sign in to download your VA education decision letter"

Comment (btruongafs) [inline src/applications/education-letters/components/LoginWidget.jsx:21]: I think this should be "Sign in to download your VA education decision letter" for the unauth/visitor user

**Review by btruongafs — APPROVED** [review]

**Review by nathanielbuck — APPROVED** [review]

**Review by apisandipas — APPROVED** [review]

**Pull request merged**